### DISCUSSION TITLE
伊埃斯真可爱嘿嘿嘿嘿嘿

### DISCUSSION THREAD

**Discussion by ShangHuaWenYuan** [share121/inter-knot#369] [其他]
有人知道哪里可以买到同款造型的伊埃斯吗？不管多少丁尼我都乐意出！！！
![4a1f8a2868f7216ad0476d5858c94e3d](https://github.com/user-attachments/assets/b093311d-dd9b-4a98-b2a8-2242d4cdd644)

**Comment by ShangHuaWenYuan** [comment]
[点❤我❤看❤可❤爱❤伊❤埃❤斯](https://www.bilibili.com/video/BV18E4m1d7fE/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by ShangHuaWenYuan** [comment]
[一❤个❤人❤在❤家❤看❤伊❤埃❤斯❤热❤舞](https://www.bilibili.com/video/BV1HT421k7Rp/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by lbghhj** [comment]
其实是你吧格丽丝，别装了
